hudi-common/src/main/java/org/apache/hudi/common/config/HoodieMetadataConfig.java

```diff
@@ -41,6 +41,15 @@
         + " to avoid overhead of accessing cloud storage, during queries.")
 public final class HoodieMetadataConfig extends HoodieConfig {
 
+  // Asynchronous cleaning for metadata table is disabled by default
+  public static final boolean DEFAULT_METADATA_ASYNC_CLEAN = false;
+  // Full scanning of log files while reading log records is enabled by default for metadata table
+  public static final boolean DEFAULT_ENABLE_FULL_SCAN_LOG_FILES = true;
+  // Meta fields are not populated by default for metadata table
+  public static final boolean DEFAULT_POPULATE_META_FIELDS = false;
+  // Default number of commits to retain, without cleaning, on metadata table
+  public static final int DEFAULT_CLEANER_COMMITS_RETAINED = 3;
```

Comment:
can we prefix "DEFAULT_METADATA" to all variables like how we have done for DEFAULT_METADATA_ASYNC_CLEAN. when used in other places, when variable alone is used, its not very apparent whether its data table defalt or MDT default